docs/core/diagnostics/dumps.md

```diff
@@ -1,79 +1,47 @@
-|---------|-----|
-|%%|A single % character|
-|%p|PID of dumped process|
-|%e|The process executable filename|
-|%h|Host name return by `gethostname()`|
-|%t|Time of dump, expressed as seconds since the Epoch, 1970-01-01 00:00:00 +0000 (UTC)|
-
-The following table shows all the values you can use for `DOTNET_DbgMiniDumpType`. For example, setting `DOTNET_DbgMiniDumpType` to 1 means `MiniDumpNormal` type dump will be collected on a crash.
-
-|Value|Name|Description|
-|-----|----|-----------|
-|1|`Mini`|A small dump containing module lists, thread lists, exception information, and all stacks.|
-|2|`Heap`|A large and relatively comprehensive dump containing module lists, thread lists, all stacks, exception information, handle information, and all memory except for mapped images.|
-|3|`Triage`|Same as `Mini`, but removes personal user information, such as paths and passwords.|
-|4|`Full`|The largest dump containing all memory including the module images.|
-
-### Collect dumps at a specific point in time
+## Analyze dumps
 
-You may want to collect a dump when the app hasn't crashed yet. For example, if you want to examine the state of an application that seems to be in a deadlock, configuring the environment variables to collect dumps on crash will not be helpful because the app is still running.
+* Navigate to [Debug Linux dumps](debug-linux-dumps.md) for information regarding analyzing dumps collected on Linux.
 
-To collect dump at your own request, you can use `dotnet-dump`, which is a CLI tool for collecting and analyzing dumps. For more information on how to use it to collect dumps with `dotnet-dump`, see [Dump collection and analysis utility](dotnet-dump.md).
+* Navigate to [Debug Windows Dumps](debug-windows-dumps.md) for information regarding analyzing dumps collected on Windows.
 
-## Analyze dumps
+### Memory Analysis
 
-You can analyze dumps using the [`dotnet-dump`](dotnet-dump.md) CLI tool or with [Visual Studio](/visualstudio/debugger/using-dump-files).
+You can perform memory analysis on your application if your app's memory continues to grow, but you are unsure why that is the case. [debug memory leak tutorial](debug-memory-leak.md) shows how to debug a memory leak using the dotnet CLI tools with the dotnet-sos commands dumpheap and gcroot.
 
-> [!NOTE]
-> Visual Studio version 16.8 and later allows you to [open Linux dumps](https://devblogs.microsoft.com/visualstudio/linux-managed-memory-dump-debugging/) generated on .NET Core 3.1.7 or later.
-> [!NOTE]
-> If native debugging is necessary, the [SOS debugger extension](sos-debugging-extension.md) can be used with [LLDB on Linux and macOS](debug-linux-dumps.md#analyze-dumps-on-linux). SOS is also supported with [Windbg/cdb](/windows-hardware/drivers/debugger/debugger-download-tools) on Windows, although Visual Studio is recommended.
+>[!NOTE]
+> [Visual Studio Memory Analysis](/visualstudio-docs/docs/profiling/analyze-memory-usage.md) can be used to diagnose a memory leak.
```

Comment:
Using a NOTE gives me the idea that the VS option is perhaps 2nd class or less recommended. For folks on Windows I expect they will probably prefer VS. Can we change this to normal text and mention that it is for Windows only?

Reply:
Can it find a memory leak in a Linux dump?

Reply:
Right now I don't think it can, but I asked the Sireesha on VS to be certain and I am waiting to hear back.